invenio_app_ils/config.py

```diff
@@ -225,8 +225,8 @@ def _(x):
 REST_CSRF_ENABLED = True
 
 # change this only while developing
```

Comment:
`CORS_SUPPORTS_CREDENTIALS` also needs to be reverted, maybe we can point to the instructions with this comment
```suggestion
# change this only while developing by creating invenio.cfg, read more https://invenioils.docs.cern.ch/quickstart/#invenio-backend
```

Reply:
I forgot this was staged on my personal machine.